input_parser.star

```diff
@@ -125,7 +128,7 @@ DEFAULT_L1_ARGS = {
     #   - apache
     #  - tracoor
     # Check the ethereum-package for more details: https://github.com/ethpandaops/ethereum-package
-    "l1_additional_services": [],
+    "l1_additional_services": ["assertoor"],
```

Comment:
I don't think this should be enabled by default. But I'd be okay if we enable it in the `.github/tests/combinations` along with the tx spammer.